<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleEditWidget</class>
 <widget class="QWidget" name="RuleEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="ruleEditLayout" stretch="0,0,0,0,1,0,0,0">
   <item>
    <widget class="QLabel" name="categoryNameLabel">
     <property name="text">
      <string>Category name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="categoryNameTextEdit"/>
   </item>
   <item>
    <widget class="Lvk::FE::RuleInputWidget" name="ruleInputWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chatbotRepliesLabel">
     <property name="text">
      <string>Chatbot replies:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Lvk::FE::RuleOutputWidget" name="ruleOutputWidget">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nextCategoryLayout">
     <item>
      <widget class="QLabel" name="nextCategoryLabel">
       <property name="text">
        <string>Jumps to category:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nextCategoryList">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QToolButton" name="undoRuleButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Undo changes</string>
       </property>
       <property name="icon">
        <iconset resource="../res/chatbot_theme.qrc">
         <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="teachRuleButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Teach rule to the chatbot</string>
       </property>
       <property name="icon">
        <iconset resource="../res/chatbot_theme.qrc">
         <normaloff>:/icons/teach_rule.png</normaloff>:/icons/teach_rule.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lvk::FE::RuleOutputWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>front-end/ruleoutputwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Lvk::FE::RuleInputWidget</class>
   <extends>QWidget</extends>
   <header>front-end/ruleinputwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/chatbot_theme.qrc"/>
 </resources>
 <connections/>
</ui>
